dataclass_rest/parse_func.py

```diff
@@ -54,15 +54,15 @@ def get_file_params(spec):
 def parse_func(
         func: Callable,
         method: str,
-        url_template: str,
+        url_template: Callable[..., str],
         additional_params: Dict[str, Any],
         is_json_request: bool,
         body_param_name: str,
 ) -> MethodSpec:
     spec = getfullargspec(func)
-    url_params = get_url_params(url_template)
     file_params = get_file_params(spec)
-    skipped_params = url_params + file_params + [body_param_name]
+    skipped_params = []
```

Comment:
why?

Reply:
because we can use function for getting url only when we has access to arguments and its values that passed to handler

like in boundmethod
```
    def _get_query_params(self, args) -> Any:
        url_params = get_url_params(self.method_spec.url_template, args)
        skipped_params = url_params + self.method_spec.file_param_names + [self.method_spec.body_param_name]
        query_params_type = create_query_params_type(getfullargspec(self.method_spec.func), self.method_spec.func, skipped_params)

        return self.client.request_args_factory.dump(
            args, query_params_type,
        )
```